ci-operator/step-registry/optional-operators/configure-realtime-tuned-profile/optional-operators-configure-realtime-tuned-profile-commands.sh

```diff
@@ -71,21 +91,34 @@ spec:
       initrd_dst_img=
       initrd_add_dir=
       # overrides cpu-partitioning cmdline
-      cmdline_cpu_part=+nohz=on rcu_nocbs=2-7 tuned.non_isolcpus=00000011 intel_pstate=disable nosoftlockup
-      cmdline_realtime=+tsc=nowatchdog intel_iommu=on iommu=pt isolcpus=managed_irq,2-7 systemd.cpu_affinity=0-1
-      cmdline_additionalArg=+ nmi_watchdog=0 audit=0 mce=off processor.max_cstate=1 idle=poll intel_idle.max_cstate=0
+      cmdline_cpu_part=+nohz=on rcu_nocbs=\${isolated_cores} tuned.non_isolcpus=\${not_isolated_cpumask} intel_pstate=disable nosoftlockup
+      cmdline_realtime=+tsc=nowatchdog intel_iommu=on iommu=pt isolcpus=managed_irq,\${isolated_cores} systemd.cpu_affinity=\${not_isolated_cores_expanded}
+      cmdline_additionalArg=+ nmi_watchdog=0 audit=0 mce=off processor.max_cstate=1 idle=poll intel_idle.max_cstate=0 nohz_full=\${isolated_cores}
     name: openshift-node-performance-manual
   recommend:
   - machineConfigLabels:
-      machineconfiguration.openshift.io/role: worker
+      machineconfiguration.openshift.io/role: ${node_role}
     operand:
       debug: false
     priority: 20
     profile: openshift-node-performance-manual
 EOF
 
-oc wait mcp/worker --for condition=Updating --timeout=5m
-oc wait mcp/worker --for condition=Updated --timeout=30m
+echo "waiting for mcp/${node_role} condition=Updating timeout=5m"
+oc wait mcp/${node_role} --for condition=Updating --timeout=5m
+
+# When applying this configuration to the master node for single node
+# we need to wait for the master node to restart before we attempt to call out to the API Server
+if [[ "$node_role" == "master" ]]; then
```

Comment:
That's a good point, that shouldn't happen here, since the machine config pool would go in `Updating` condition first before the MCO triggers a restart, this should also catch if it ever fails to go into `Updating` then we would want it to fail fast. That's how it's been behaving when I've ran this script on SNO deployments on another PR to test things out